frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -205,7 +208,7 @@ const SampleDetailsContent = ({
       {error &&
         <ErrorMessage error={error} />
       }
-      <Tabs activeKey={activeKey} onChange={setActiveKey} size="large" type="card" style={tabsStyle}>
+      <Tabs activeKey={activeKey} onChange={setActiveKey} size="large" type="card" className={tabsClass}>
```

Comment:
You only need to add your new class name here:
`className='lineage-tab-active'`
It will add your new class name to the existing ant class names (note that you don't include the dot here).

There's no need to use the classNames package for this.